### PR TITLE
Stories and controls for new survey responses

### PR DESCRIPTION
**Pull request by mehalshah**
image

image

### CODE REVIEW

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <table>
+          <tbody>
+          {
+            this.props.possibleAnswers.map((possibleAnswer, i) => {
+              let answerIndex;
+
+              if (this.props.answerType === 'selectValue') {
+                answerIndex = i + 1;
+              } else {
+                answerIndex = possibleAnswer;
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx]: I'm confused by this `answerIndex`. How is the index the same as the `possibleAnswer` for non-value questions? Won't that be text in all other cases?

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import _ from 'lodash';
+import {Panel} from 'react-bootstrap';
+
+export default class SingleChoiceResponses extends React.Component {
+  static propTypes = {
+    question: PropTypes.string.isRequired,
+    answers: PropTypes.object.isRequired,
+    answerType: PropTypes.string.isRequired,
+    possibleAnswers: PropTypes.array.isRequired,
+    otherAnswers: PropTypes.array
```
Comment (aoby) [inline apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx]: What is `otherAnswers`? Is that a collection of all filled-in "other" responses to this question? Please add some description here for the component and its props.

Comment (clareconstantine) [reply apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx]: +1 a brief explanation of `answers` as opposed to `otherAnswers`

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx (hunk 1 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx]: For selectValue questions, the value is not the text of the response but a numeric value in some range. The hash contains a map of the numeric values to counts, not texts to counts.

(I'll add this comment)

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results/text_responses.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        answers.unshift((
+          <li>
+            Average: {average}
+          </li>
+        ));
+      }
+
+      return answers;
+    } else {
+      return (<span>nope</span>);
```
Comment (clareconstantine) [inline apps/src/code-studio/pd/workshop_dashboard/components/survey_results/text_responses.jsx]: this doesn't look like it should be here...

**Pull request merged**